azkaban-exec-server/src/test/java/azkaban/execapp/event/RemoteFlowWatcherTest.java

```diff
@@ -217,10 +203,10 @@ private void testPipelineLevel2(final ExecutableFlow first, final ExecutableFlow
         Assert.assertEquals(child.getStatus(), Status.SUCCEEDED);
         final long diff = node.getStartTime() - child.getEndTime();
         minDiff = Math.min(minDiff, diff);
```

Comment:
This logic is duplicated in the same class. It would be great if they can be consolidated. It can be a separate PR though.